src/content/docs/devtools/Figma/figma-video-case-study.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Figma Video Case Study
+author: Andreea Bianca Simon <absim29>
+tags: figma, case study, platform, video, design, collaboration, UI/UX
+---
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/OiAMaEWy2qA?si=KO0ji_hM81Dsb1ym" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
great video! nice research only one negative conclusion. time 2.13 (around) i would change a background, because is disturbing a user to focus